src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html

```diff
@@ -37,14 +37,14 @@
                 <div class="row g-0 mb-1" *ngIf="complaintsEnabled">
                     <div class="col-7">{{ 'artemisApp.assessmentDashboard.complaints' | artemisTranslate }}</div>
                     <div class="col-5">
-                        <div *ngIf="isAtLeastInstructor">
+                        <div *ngIf="course?.isAtLeastInstructor">
                             <a id="open-complaints" [routerLink]="complaintsLink"> {{ complaints.done }} / {{ complaints.total }} ({{ complaints.doneToTotalPercentage }}) </a>
                             |
                             <a [routerLink]="complaintsLink" [queryParams]="{ tutorId: tutorId }">
                                 {{ complaints.tutor }}
                             </a>
                         </div>
-                        <div *ngIf="!isAtLeastInstructor">
+                        <div *ngIf="!course?.isAtLeastInstructor">
```

Comment:
This would evaluate to true of the course is not defined! Please check if this could cause any problems.

Reply:
Good catch. I also improved such occurrences in three other files